<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FPagoBase</class>
 <widget class="QWidget" name="FPagoBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>952</width>
    <height>817</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>750</width>
    <height>550</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Formas de pago</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../../bulmages.qrc">
    <normaloff>:/Images/payment-method.png</normaloff>:/Images/payment-method.png</iconset>
  </property>
  <layout class="QGridLayout">
   <property name="margin">
    <number>5</number>
   </property>
   <property name="spacing">
    <number>5</number>
   </property>
   <item row="1" column="1">
    <widget class="QGroupBox" name="groupBox1">
     <property name="title">
      <string>Informacion de la forma de pago:</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_1">
        <property name="text">
         <string>Nombre de la forma de pago:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="mui_descforma_pago">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>0</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Plazo entre recibos (dias):</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="mui_plazoEntreReciboFPago">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_3">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Numero de plazos:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="mui_nPlazosFPago">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_4">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Descuento:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="mui_descuentoforma_pago"/>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_5">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Plazo primer recibo (dias):</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="mui_dias1tforma_pago"/>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_6">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Tipo plazo entre recibos:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLineEdit" name="mui_tipoPlazoEntreReciboFPago">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="label_7">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Tipo plazo primer pago:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QLineEdit" name="mui_tipoPlazoPrimerPago">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="label_8">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Banco:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="BfBuscarBanco" name="mui_idbanco"/>
      </item>
      <item row="8" column="0" colspan="2">
       <widget class="QWidget" name="mui_plugin_frame" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>2</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="9" column="0" colspan="2">
       <widget class="QFrame" name="m_frameplugin">
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Plain</enum>
        </property>
       </widget>
      </item>
      <item row="10" column="0" colspan="2">
       <spacer>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Expanding</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>387</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Forma de pago:</string>
     </property>
     <layout class="QGridLayout">
      <property name="margin">
       <number>4</number>
      </property>
      <property name="spacing">
       <number>4</number>
      </property>
      <item row="0" column="0">
       <widget class="QListWidget" name="mui_lista"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <layout class="QHBoxLayout">
     <property name="spacing">
      <number>4</number>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="label">
       <property name="minimumSize">
        <size>
         <width>48</width>
         <height>48</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>48</width>
         <height>48</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="../../../bulmages.qrc">:/Images/payment-method.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QFrame" name="frame">
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QGridLayout">
        <property name="margin">
         <number>4</number>
        </property>
        <property name="spacing">
         <number>4</number>
        </property>
        <item row="0" column="0">
         <widget class="QToolButton" name="mui_crear">
          <property name="minimumSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Nueva forma de pago</string>
          </property>
          <property name="whatsThis">
           <string>Nueva forma de pago</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../../../bulmages.qrc">
            <normaloff>:/Images/document-new.png</normaloff>:/Images/document-new.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>22</width>
            <height>22</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QToolButton" name="mui_guardar">
          <property name="minimumSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Borrar forma de pago</string>
          </property>
          <property name="whatsThis">
           <string>Borrar forma de pago</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../../../bulmages.qrc">
            <normaloff>:/Images/edit-save.png</normaloff>:/Images/edit-save.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>22</width>
            <height>22</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QToolButton" name="mui_borrar">
          <property name="minimumSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Guardar forma de pago</string>
          </property>
          <property name="whatsThis">
           <string>Guardar forma de pago</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>:Images/edit-delete.png</normaloff>:Images/edit-delete.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>22</width>
            <height>22</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <spacer>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>300</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0" colspan="2">
    <layout class="QHBoxLayout">
     <property name="spacing">
      <number>4</number>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="mui_aceptar">
       <property name="text">
        <string>&amp;Aceptar</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="mui_cancelar">
       <property name="text">
        <string>&amp;Cancelar</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>BfBuscarBanco</class>
   <extends>QComboBox</extends>
   <header>bfbuscarbanco.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../../bulmages.qrc"/>
  <include location="../../../bulmages.qrc"/>
 </resources>
 <connections/>
</ui>
